src/components/asset-list/RecyclerAssetList2/profile-header/ProfileActionButtonsRow.tsx

```diff
@@ -295,31 +299,28 @@ function MoreButton() {
 
   const { mostRecentWalletConnectors } = useWalletConnectConnections();
 
-  const menuConfig = React.useMemo(
-    () => ({
-      menuItems: [
-        {
-          actionKey: 'copy',
-          actionTitle: lang.t('wallet.copy_address'),
-          icon: { iconType: 'SYSTEM', iconValue: 'doc.on.doc' },
-        },
-        {
-          actionKey: 'qrCode',
-          actionTitle: lang.t('button.my_qr_code'),
-          icon: { iconType: 'SYSTEM', iconValue: 'qrcode' },
-        },
-        mostRecentWalletConnectors.length > 0
-          ? {
-              actionKey: 'connectedApps',
-              actionTitle: lang.t('wallet.connected_apps'),
-              icon: { iconType: 'SYSTEM', iconValue: 'app.badge.checkmark' },
-            }
-          : null,
-      ].filter(Boolean),
-      ...(ios ? { menuTitle: '' } : {}),
-    }),
-    [mostRecentWalletConnectors.length]
-  );
+  const menuConfig = {
```

Comment:
Noticed this sometimes didn't update, so just removed the memo since it's not very perf intensive.